packages/nextjs/src/config/wrappers/withSentryGetServerSideProps.ts

```diff
@@ -16,9 +17,24 @@ export function withSentryGetServerSideProps(
   return async function (
     ...getServerSidePropsArguments: Parameters<GetServerSideProps>
   ): ReturnType<GetServerSideProps> {
-    return callDataFetcherTraced(origGetServerSideProps, getServerSidePropsArguments, {
-      parameterizedRoute,
-      dataFetchingMethodName: 'getServerSideProps',
-    });
+    const [context] = getServerSidePropsArguments;
+    const { req, res } = context;
+
+    const errorWrappedGetServerSideProps = withErrorInstrumentation(origGetServerSideProps);
+
+    if (hasTracingEnabled()) {
+      return callServerSideDataFetcherWithTracingInstrumentation(
+        errorWrappedGetServerSideProps,
+        getServerSidePropsArguments,
+        req,
+        res,
+        {
+          parameterizedRoute,
+          functionName: 'getServerSideProps',
```

Comment:
Suggestion: Read this off of the function itself, inside the helper, rather than passing it.

Reply:
Honestly, I don't see the upside of doing that. We know it's `getServerSideProps` so let's just tell the wrapper that it is. No need to do some function name reading magic.

Reply:
I mean... if you consider `origFunction.name` magic... 😛